anaUltraScurve.py

```diff
@@ -284,44 +345,49 @@ def overlay_fit(VFAT, CHAN):
         pass
     pass
 
-canv = r.TCanvas('canv','canv',500*8,500*3)
-legend = r.TLegend(0.75,0.7,0.88,0.88)
-if not options.PanPin:
-    canv.Divide(8,3)
-    r.gStyle.SetOptStat(0)
-    for vfat in range(0,24):
+def saveSummary(vSum, vSum2, name='Summary'):
```

Comment:
Line 348, why not add the function "saveSummary()" to "anautilities.py" and add arguments for it taking input lists of histograms as arguments.

Seems like:
> anaUltraScurve.py
> anaUltraThreshold.py
> anaUltraLatency.py

Could all make use of this and this could also be used by future scripts.

Functionality should be user supplies list of histograms as an argument indexed by VFAT position, then saveSummary() will just make a single output canvas in the standard 3x8 (row x col) plot.  Then this function could be called for each list of histograms.